doc/introduction/dynamic_quantum_circuits.rst

```diff
@@ -433,17 +431,18 @@ As described above, there are multiple simulation techniques for circuits with
 mid-circuit measurements in PennyLane. They can be configured when initializing a
 :class:`~pennylane.QNode`, using the following keywords:
 
-* ``mcm_method``: Sets the method used for applying mid-circuit measurements. The options are
-  ``"deferred"``, ``"one-shot"``, and ``"tree-traversal"`` for the three techniques described above.
-  The default is ``mcm_method="one-shot"`` when executing with shots, and ``"deferred"`` otherwise.
-  When using :func:`~pennylane.qjit`, there is the additional (default) option
-  ``mcm_method="single-branch-statistics"``, which explores a single branch of the execution
-  tree at random.
-
-  .. warning::
-
-      If the ``mcm_method`` argument is provided, the transforms for deferred measurements
-      or dynamic one-shot sampling must not be applied manually to the :class:`~pennylane.QNode`.
+* `mcm_method`: Sets the method used for applying mid-circuit measurements.
+  The three techniques described above can be specified with `"deferred"`,
+  `"one-shot"`, and `"tree-traversal"`. When using :func:`~pennylane.qjit`,
+  there is the additional (default) option `"single-branch-statistics`, which
```

Comment:
```suggestion
  The three techniques described above can be specified with ``"deferred"``,
  ``"one-shot"``, and ``"tree-traversal"``. When using :func:`~pennylane.qjit`,
  there is the additional (default) option ``"single-branch-statistics"``, which
```

On top of Andrija's suggestion 👀

Reply:
Also, we're planning on having one-shot be default with Catalyst: https://app.shortcut.com/xanaduai/epic/65655

@paul0403 is this still on track for this quarter? If so, @astralcai I think we can safely change the above and below paragraphs to say that "one-shot" is default with qjit